Name: Enable USB Passthrough
Version: 1.3.5-1
Author: Ebag333
Description: Removes the required code for USB Passthrough and adds the icon to the launcher.

--- .orig/usr/palm/applications/com.palm.app.usbpassthrough/app/controllers/usbpassthrough-assistant.js
+++ /usr/palm/applications/com.palm.app.usbpassthrough/app/controllers/usbpassthrough-assistant.js
@@ -176,8 +176,8 @@
 			
 			if (this.isActivationFile != true) {
 				$('masterGroup').style.display = 'none';
-				$('menuGroup').style.display = 'none'; // show menu
-				$('entryGroup').style.display = 'block';
+				$('menuGroup').style.display = 'block'; // show menu
+				$('entryGroup').style.display = 'none';
 			}
 		} catch (e) {
 			Mojo.Log.error ("main.activate(): exception: " + e)
--- .orig/usr/palm/applications/com.palm.app.usbpassthrough/appinfo.json
+++ /usr/palm/applications/com.palm.app.usbpassthrough/appinfo.json
@@ -6,7 +6,7 @@
 	"icon": "icon.png",
 	"version": "1.0.27",
 	"noWindow": false,
-	"visible": false,
+	"visible": true,
 	"vendor": "Palm, Inc."
 }
 
